cmd/api/src/queries/graph.go

```diff
@@ -337,34 +362,52 @@ func formatSearchResults(exactResults []model.SearchResult, fuzzyResults []model
 	return searchResults[skip:end]
 }
 
-func (s *GraphQuery) SearchNodesByName(ctx context.Context, nodeKinds graph.Kinds, name string, skip int, limit int) ([]model.SearchResult, error) {
+type NodeSearchResults struct {
+	ExactResults []model.SearchResult
+	FuzzyResults []model.SearchResult
+}
+
+func (s *GraphQuery) SearchNodesByNameOrObjectId(ctx context.Context, nodeKinds graph.Kinds, nameOrObjectId string, skip int, limit int) ([]model.SearchResult, error) {
```

Comment:
NOT FOR THIS PR: for searching across two fields we should consider a trigram index for `name` and `objectid`. Not sure how this would work in Neo4J but for Postgres:

``` pgsql
CREATE INDEX node_objectid_name_trgm_idx ON nodes USING GIN (CONCAT_WS(' ', objectid, name) gin_trgm_ops)

SELECT *
FROM nodes
WHERE CONCAT_WS(' ', objectid, name) % 'some parameterized search string';
```

Consider capturing as a research spike for backlog.

Reply:
Spike ticket created! https://specterops.atlassian.net/browse/BED-6882